nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-avro-record-utils/src/test/resources/org/apache/nifi/avro/schema.json

```diff
@@ -1,20 +1,20 @@
 [ {
-	"namespace" : "net.a",
-	"type" : "record",
-	"name" : "O",
-	"fields" : [ {
-		"name" : "hash",
-		"type" : [ "null", {
-			"type" : "map",
-			"values" : "string"
-		} ]
-	} ]
-}, {
 	"namespace" : "net.a",
 	"type" : "record",
 	"name" : "A",
 	"fields" : [ {
 		"name" : "o",
-		"type" : [ "null", "O" ]
+		"type" : [ {

```

Comment:
This appears to have changed with the parsing functionality in 1.9+. Under 1.10 and 1.11, the Avro API was not able to the first type and reference it in the second type until it was embedded in the second type as shown here.